script/database_role/README.md

```diff
@@ -0,0 +1,20 @@
+# Terraform Module: DB SQL Script - Create readonly role
+
+Run SQL script in DB to create DB role(s).
+This script will be run on the machine which run this TF script.
+
+## Usage
+
+```terraform
+module "db_readonly_script" {
```

Comment:
Could we use this module in the RDS module? E.g. pass as a variable sth like `read_only_role: Boolean` (default `true`?) to create the read-only role when we create the DB instance?

this will reduce the amount of extra steps for each new stack.

Reply:
I think this would not reduce the number of extra steps because we still need to run the script afterwards. eg.
1. stack/app (for creation of ecs <bastion> and rds)
2.  deploy bastion
3. Run db script

Reply:
Could the RDS module output a reminder to run this?